README.md

```diff
@@ -20,15 +20,15 @@ $ wget https://github.com/conda-forge/miniforge/releases/latest/download/Minifor
 # For MacOS:
 # $ wget https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-x86_64.sh
 
-$ bash ./Miniforge3-Linux-x86_64.sh
+$ sh ./Miniforge3-Linux-x86_64.sh
 # make sure the installer runs 'conda init' at the end of installation, or run it manually.
 
 # Restart shell to load conda integration
-$ exec bash
+$ exec $SHELL
 
 # Optional: create conda environment
-$ conda create -n dgfem
-$ conda activate dgfem
+$ $CONDA_EXE create -n dgfem
+$ $CONDA_EXE activate dgfem
```

Comment:
I don't find that super helpful: Why would a user know what this should be set to?

Reply:
Isn't CONDA_EXE set automatically (see e.g. https://github.com/conda/conda/issues/7126)?

Reply:
I don't see how this could be set by a subprocess in a running shell. I also just tried it, and after running `miniforge*.sh`, `$CONDA_EXE` is not defined.

Reply:
ok, changed to explicit `export`.